luma/core/cmdline.py

```diff
@@ -121,7 +121,7 @@ def load_config(path):
     return args
 
 
-class make_serial(object):
+class make_interface(object):
```

Comment:
this is a backward incompat change. Can you add a `make_serial = make_interface` with a comment it's deprecated?

Reply:
done